workshop/working-with-your-own-data.md

```diff
@@ -107,7 +107,7 @@ wget -O data/some_array_data.zip 'https://www.ebi.ac.uk/arrayexpress/files/E-GEO
 `-O` is one of many `wget` command options.
 To see the complete list of `wget` options, use the command: `wget -h` in Terminal.
 
-[See more `wget` examples](https://www.tecmint.com/10-wget-command-examples-in-linux/)
+You can see some more `wget` examples [at this link](https://www.tecmint.com/10-wget-command-examples-in-linux/).
```

Comment:
You may have heard that I don't like "here" links. This isn't quite that, but the link text in my mind should still be "more `wget` examples".